src/core/core.js

```diff
@@ -13,7 +13,7 @@ module.exports = function() {
 		};
 
 		// Support a jQuery'd canvas element
-		if (context.length && context[0].getContext) {
+		if (typeof(jQuery) === 'function' && context instanceof jQuery) {
```

Comment:
Will this continue to work even if jQuery is not in the global namespace? I think that case could happen in a commonJS style build.

I also think we should, if possible, add a test for this